docs/dietpi_sbc.md

```diff
@@ -276,7 +276,8 @@ Careful consideration went into the design and development of the Tinker Board t
 ## NanoPi Series (FriendlyARM)
 
 **FriendlyARM's next generation SBC** - One of the fastest ARM SBC (RK3399) on the market today, with 2x1.8GHz A72 cores and 4x1.4GHz A53 cores. One of the smallest RK3399 SBCs available, a great addition to your SBC lineup that provides next gen SBC performance.
-For more details visit the [FriendlyARM](https://www.friendlyarm.com/) website.
+
+For more details visit **FriendlyARM** website.
```

Comment:
This link to friendlyarm.com failed often at the commit check (at least after the daylight saving switchover).

Reply:
Hmm, since it does generally work, I would still keep it. I mean the Twitter link for some reason also fails regularly, I think we simply need to accept that this check has false negatives sometimes. I enabled verbose print so that all positive link checks are shown as well, mostly to verify/review that all links are checked, but I can remove that so that only the failed links are listed which allows to quickly evaluate whether something needs to be fixed or it is a known often failing candidate.

Reply:
Changed it back. I aggree to Micha.